nautobot/docs/models/extras/configcontext.md

```diff
@@ -11,6 +11,7 @@ Sometimes it is desirable to associate additional data with a group of devices o
 * Cluster
 * Tenant group
 * Tenant
+* Device redundancy groups
```

Comment:
Is the plural here accurate? I asked since all the other `foo group`/ all the other items on this list are singular.

Reply:
```suggestion
* Device redundancy group
```